packages/EasyDecision/src/Decisions/DecisionFactory.php

```diff
@@ -47,6 +53,7 @@ final class DecisionFactory implements DecisionFactoryInterface
      * @param null|iterable<mixed> $configurators
      */
     public function __construct(
+        DecisionAggregatorInterface $decisionAggregator,
```

Comment:
Like I said on Slack, nice try, I've thought about doing the same thing in other packages but if you're adding a brand new service only for Symfony DataCollector then simply just modify the existing service to expose the data 😄 

So I don't think we need this aggregator, simply have a public method to get the list of configured decisions, inject this decision factory in the data collector and that's it!